### PR TITLE
Fix indirect imports

### PR DESCRIPTION
**Pull request by sethfischer**
Fix two indirect imports which makes the `cq_directive.py` module easier to follow.

### CODE REVIEW

**Review by jmwright — APPROVED** [review]
Thanks @sethfischer 

We'll wait for one other review before merging, but I think it looks good.

**Review by adam-urbanczyk — APPROVED** [review]

**Comment by adam-urbanczyk** [comment]
LGTM, thanks @sethfischer

**Pull request merged**
